### PR TITLE
Dockerise nbsap

### PR DESCRIPTION
**Pull request by anaion**
PR for #26735.

@alexef please review

### CODE REVIEW

File: docker-compose.prod.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+ch:
+    image:nbsap_web
+    volumes:
+        - ./instance_ch/:/nbsap/instance
+    ports:
+        - "8000:7012"
+    links:
+        - db
+    env_file:
+        - ./env_files/ch.env
```
Comment (alexef) [inline docker-compose.prod.yml:24]: Why have both instance_<country> and <country>.env?

Can you please add a country setup example?

**Comment by alexef** [comment]
Please add a README describing the steps to set-up a new set of instances on a single host docker instance.

File: docker-compose.prod.yml (hunk 1 above)
Comment (anaion) [inline docker-compose.prod.yml:24]: Which .env?

instance_\* overrides the container's instance, why should it contain any Docker environment variables?

What do you mean by country setup? An env file?

Comment (alexef) [inline docker-compose.prod.yml:24]: On a second thought, the `instance_*` folders/files should be added to this repo, the only part to be kept secret being the `.env` files, containing database credentials.

Right now, we have the orchestration without essential files for a developer to configure and start an environment similar to the current production server.

Comment (anaion) [inline docker-compose.prod.yml:24]: Sorry, but I still don't get it. Which are the essential files you are referring to? Per instance env files? They will contain sensitive information, so why should they be pushed to the repo? The dev.env stands as an example on how they should look like.

Comment (alexef) [inline docker-compose.prod.yml:24]: I'm talking about per instance `local_settings.py` files which contain non sensitive information such as: instance title and instance languages.

**Pull request merged**